exercises/practice/roman-numerals/roman_numerals.pony

```diff
@@ -0,0 +1,9 @@
+primitive RomanNumerals
+  fun apply(number: USize): String ? =>
+    """
+    Convert the number to a roman number.
```

Comment:
I do not know if this is generated from or hand written, but I think they are referred to as Roman numerals.  So proper name, and instead of number, numeral.